circle.yml

```diff
@@ -18,18 +21,28 @@ dependencies:
     # node_modules is included by default
 
   override:
+    - pip install tox mozdownload mozinstall
     - google-chrome --version
     - ./bin/circleci/install-node-dependencies.sh
-    - ./bin/circleci/setup-test-dependencies.sh
     - ./bin/circleci/build-addon.sh
 
 test:
+  pre:
+    - sudo apt-get upgrade firefox
+    - wget -O geckodriver.tar.gz https://github.com/mozilla/geckodriver/releases/download/v0.16.0/geckodriver-v0.16.0-linux64.tar.gz
```

Comment:
I recently updated the tests for pytest-selenium to always get the latest GeckoDriver release. You could potentially do something similar: https://github.com/pytest-dev/pytest-selenium/blob/master/.travis.yml#L30-L31